homeassistant/components/vicare/strings.json

```diff
@@ -0,0 +1,24 @@
+{
+    "config": {
+        "flow_title": "{name} ({host})",
+        "step": {
+            "user": {
+                "title": "{name}",
+                "description": "Set up ViCare integration. To generate API key go to https://developer.viessmann.com",
+                "data": {
+                    "username": "[%key:common::config_flow::data::email%]",
+                    "password": "[%key:common::config_flow::data::password%]",
+                    "client_id": "[%key:common::config_flow::data::api_key%]",
+                    "heating_type": "Heating type"
+                }
```

Comment:
We're still asking the user for name and scan interval. Until we remove them from the form dataschema we need strings for them.

https://github.com/home-assistant/core/blob/177ffa3aa67c856545022b13ca63f1b3e93f8985/homeassistant/components/vicare/config_flow.py#L46-L57

Reply:
Right. I'll add them
I'd prefer to rework the integration according to your comments for the next release.